catalogue/webapp/pages/search/index.tsx

```diff
@@ -206,27 +200,27 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     const defaultProps = removeUndefinedProps({
       serverData,
       query,
-
       pageview: {
         name: 'search',
         properties: {},
       },
     });
 
-    // Stop here if no query has been entered
-    if (!params.query) {
-      return {
-        props: defaultProps,
-      };
-    }
-
     try {
       // Stories
+      // We want the default order to be "descending publication date"
```

Comment:
Can we explain why we want this to be the default order?

Reply:
Unsure how to explain it apart from "to advertise the latest posts" but it feels obvious?